app/build.gradle

```diff
@@ -398,6 +398,8 @@ dependencies {
     implementation ('com.adamratzman:spotify-api-kotlin-android:3.7.0') {
         exclude group: 'org.jetbrains.kotlinx', module: 'kotlinx-coroutines-core'
     }
+    implementation "com.soywiz.korlibs.korim:korim-android:2.2.0"
```

Comment:
I suspect this was a previous direction, but seems no longer used presently

Reply:
This is kind of an interesting scenario since when I was trying to utilize the uploadClientPlaylistCover from the ClientPlaylistAPI I was running into the issue that it was failing to compile and also experiencing runtime errors due to the korim and korio dependencies not being found. I looked into this issue on the SpotifyWebApi library's discord server and turns out there is a bug/something broken around this and the developer has been busy lately so this is kind of a hacky workaround in the meantime until that fix is released 😅